src/game/game.c

```diff
@@ -13,6 +13,17 @@
 
 #include <SDL2/SDL_image.h>
 
+static void set_window_title(struct oshu_game **game)
+{
+	const char * title = "Oshu! ♫ \"%s\" by %s";
+
+	asprintf(&(*game)->display->window_title, title,
+		(*game)->beatmap->metadata.title,
+		(*game)->beatmap->metadata.artist);
+	SDL_SetWindowTitle((*game)->display->window,
+		(*game)->display->window_title);
```

Comment:
I think it's safe to assume SDL copies the string. You should free the string allocated by asprintf at the end of the function.

And since asprintf may fail, at least write an explicit *assert*.